src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -122,6 +130,11 @@ export default {
 		}
 	},
 	mounted() {
+		// hide 'new countries for you' category if user is not logged in
+		if (this.isLoggedIn === false && this.url.includes('new-countries-for-you') === true) {
+			this.showCategory = false;
+		}
```

Comment:
`showCategory` could be a computed property so that it updates if the user's logged-in state changes

Reply:
@emuvente I've updated `showCategory' to be a computed property. Correct?